src/Symfony/Component/Security/Core/Security.php

```diff
@@ -57,4 +66,39 @@ public function getToken(): ?TokenInterface
     {
         return $this->container->get('security.token_storage')->getToken();
     }
+
+    public function autoLogin(UserInterface $user, AuthenticatorInterface $authenticator = null, string $firewallName = null): void
```

Comment:
I guess we can call it `login()` for the same reasoning as the `logout()` one.

Reply:
Changed. Thank you !